core/chains/evm/logpoller/log_poller.go

```diff
@@ -28,39 +28,38 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
 	evmtypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
 	ubig "github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils/big"
-	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
 )
 
 //go:generate mockery --quiet --name LogPoller --output ./mocks/ --case=underscore --structname LogPoller --filename log_poller.go
 type LogPoller interface {
 	services.Service
 	Replay(ctx context.Context, fromBlock int64) error
 	ReplayAsync(fromBlock int64)
-	RegisterFilter(filter Filter, qopts ...pg.QOpt) error
-	UnregisterFilter(name string, qopts ...pg.QOpt) error
+	RegisterFilter(filter Filter) error
+	UnregisterFilter(name string) error
 	HasFilter(name string) bool
-	LatestBlock(qopts ...pg.QOpt) (LogPollerBlock, error)
-	GetBlocksRange(ctx context.Context, numbers []uint64, qopts ...pg.QOpt) ([]LogPollerBlock, error)
+	LatestBlock(ctx context.Context) (LogPollerBlock, error)
```

Comment:
Shouldn't the other methods swap for a `context.Context` too?

Reply:
For sure, I missed this when focusing on getting tests to pass. Just added context to all of them 👍